ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-ref.yaml

```
@@ -28,5 +28,7 @@ ref:
   - name: REGION
     default: ""
     documentation: Use a specific AWS region, overriding the LEASED_RESOURCE environment variable in the cluster_profile.
+  - name: IPSTACK
+    default: ""
```

Add a short description here?
```suggestion
    default: ""
    documentation: |-
      Valid values:
       "": (default), IPv4 only
       "dualstack": Enable IPv6.
```